docker/docker-compose.dev.yml

```
@@ -129,4 +129,4 @@ volumes:
   github_repos:
   wordlist:
   scan_results:
-  ollama_data:
+  ollama_data:
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)